WowPacketParser/Store/Objects/GossipMenuAddon.cs

```diff
@@ -0,0 +1,23 @@
+using WowPacketParser.Enums;
+using WowPacketParser.Misc;
+using WowPacketParser.SQL;
+
+namespace WowPacketParser.Store.Objects
+{
+    [DBTableName("gossip_menu_addon")]
+    public sealed record GossipMenuAddon : IDataModel
+    {
+        [DBFieldName("MenuID", true)]
+        public uint? MenuID;
+
+        [DBFieldName("FriendshipFactionID", true)]
```

Comment:
Primary key should be on menu id only or do we know of cases where one menuid has to support multiple different factionids?